lib/beaker/ssh_connection.rb

```diff
@@ -113,13 +113,15 @@ def close
     # @option options [String] :stdin Any input to be sent along with the command
     # @param [IO] stdout_callback An IO stream to send connection stdout to, defaults to nil
     # @param [IO] stderr_callback An IO stream to send connection stderr to, defaults to nil
+    # @param [Integer] poll_interval A set interval of time to poll for connectivity, defaults to nil
+    # @param [Integer] max_tries The maximum number of times to retry connection, defaults to 11
     # @return [Boolean] true if connection failed, false otherwise
-    def wait_for_connection_failure options = {}, stdout_callback = nil, stderr_callback = stdout_callback
+    def wait_for_connection_failure options = {}, stdout_callback = nil, stderr_callback = stdout_callback, poll_interval = nil, max_tries = 11
```

Comment:
1. No different reasons, just enjoy the additional intellisense in my IDE when there are actual parameters to methods.
2. It shouldn't impact any existing calls since the new params are at the end of the method and have default values.